tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -44,49 +44,56 @@ function Reports() {
 
   const currentStt = isOFAAdmin ? selectedStt : userProfileStt
 
+  const stt = sttList?.find((stt) => stt?.name === currentStt)
+  const [submittedHeader, setSubmittedHeader] = useState('')
+
+  const errorsCount = formValidation.errors
+
+  const missingStt = !isOFAAdmin && !currentStt
+
+  const errorsRef = useRef(null)
+
   const handleSearch = () => {
-    if (!isUploadReportToggled) {
-      // Clear previous errors
-      setFormValidationState({})
-      // Filter out non-truthy values
+    // Clear previous errors
+    setFormValidationState({})
+    // Filter out non-truthy values
 
-      const form = [selectedYear, currentStt, selectedQuarter].filter(Boolean)
+    const form = [selectedYear, currentStt, selectedQuarter].filter(Boolean)
+    const reportHeader = `${currentStt} - Fiscal Year ${selectedYear} - ${quarters[selectedQuarter]}`
 
-      if (form.length === 3) {
-        setIsToggled(true)
-      } else {
-        // create error state
-        setFormValidationState({
-          year: !selectedYear,
-          stt: !currentStt,
-          quarter: !selectedQuarter,
-          errors: 3 - form.length,
-        })
-        setTouched({
-          year: true,
-          stt: true,
-          quarter: true,
-        })
-      }
+    if (form.length === 3) {
+      setIsToggled(true)
+      setSubmittedHeader(reportHeader)
```

Comment:
Save a reference to the previous "submitted" header, so that the header that is currently rendered is static, and does not change if the user changes form selections.